### DISCUSSION TITLE
Error advanced media extensions (AME) after installation

### DISCUSSION THREAD

**Discussion by gildastambo** [007revad/Video_Station_for_DSM_722#16] [General]
while trying too open Video Station after the installation i get this massage : **_to use video station , please install the latest advanced media extensions (AME) and update its codec pack.  but it is already up to date_**
and when i attend to  manually updating it, i get this **_Current package  version is 30.1.0-3005. You cannot roll back the older version 4.0.0-4025...._**. do you have any idea how to solve this please?

**Comment by 007revad** [comment]
You've got Video Station 3.1.1-3168 installed.

Go to package center and uninstall Video Station 3.1.1-3168 then run the script again so it can install Video Station 3.1.0-3153

**Comment by gildastambo** [comment]
It works well now that you indeed !
source for manual installation: https://archive.synology.com/download/Package/VideoStation/3.1.0-3153

**Reply by 007revad** [reply]
The script also changes the version to **30**.1.0-3153 to prevent the annoying "newer version available" from appearing in package center.
